src/ctype/CType.chain.ts

```diff
@@ -26,7 +22,7 @@ export async function store(
   log.debug(() => `Create tx for 'ctype.add'`)
   const tx: SubmittableExtrinsic<
     CodecResult,
-    any
+    SubscriptionResult
```

Comment:
All the SubmittableExtrinsic come from the Polkadot api and the SubscriptionResult is what they used. It seemed a natural fit. It is a Promise<any>.